pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -225,6 +225,107 @@ int uretprobe__SSL_write(struct pt_regs* ctx) {
     return 0;
 }
 
+SEC("uprobe/SSL_read_ex")
+int uprobe__SSL_read_ex(struct pt_regs* ctx) {
+    ssl_read_ex_args_t args = {0};
+    args.ctx = (void *)PT_REGS_PARM1(ctx);
+    args.buf = (void *)PT_REGS_PARM2(ctx);
+    args.size_out_param = (size_t *)PT_REGS_PARM4(ctx);
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+    log_debug("uprobe/SSL_read_ex: pid_tgid=%llx ctx=%llx\n", pid_tgid, args.ctx);
+    bpf_map_update_elem(&ssl_read_ex_args, &pid_tgid, &args, BPF_ANY);
```

Comment:
What about the return value? don't we have telemetry for that?

Reply:
of `bpf_map_update_elem`?

Reply:
True

Reply:
irrelevant at the moment
it is best effort

Reply:
If you wish you can begin collecting telemetry for this map, after rebasing with main, by replacing the helper call with `bpf_map_update_with_telemetry`.